src/preferences/dialog/dlgpreflv2.cpp

```diff
@@ -87,18 +88,22 @@ void DlgPrefLV2::slotDisplayParameters() {
 
     EffectManifestPointer pCurrentEffectManifest = m_pLV2Backend->getManifest(pluginId);
     if (pCurrentEffectManifest) {
+        // Show the effect name above the parameter list
+        effect_name_label->setText(QObject::tr("Parameters for %1")
```

Comment:
camel case?

Reply:
in the preferences ui files I couldn't figure a rule.
I'll go with _ for better readability.